trying to create a new project

Notes
======
trying to create a new project (Android 4.4.2)

Environment
======
Environment Key | Value
:----: | :----:
version | 11.0
build | 164
UDID | f710314c4d27668b
Android Release | 4.4.2
Android SDK | 19
Brand | rockchip
Device | rk292x
Stack trace
======
```java
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.translationstudio.androidapp/com.door43.translationstudio.ui.translate.TargetTranslationActivity}: java.lang.NullPointerException
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2195)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2245)
	at android.app.ActivityThread.access$800(ActivityThread.java:135)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1196)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:136)
	at android.app.ActivityThread.main(ActivityThread.java:5017)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:515)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:795)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:611)
	at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.NullPointerException
	at com.door43.translationstudio.ui.translate.TargetTranslationActivity.onCreate(TargetTranslationActivity.java:222)
	at android.app.Activity.performCreate(Activity.java:5231)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1087)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2159)
	... 11 more

```
Log history
======
```java
3/22/17 5:21 PM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation en-x-demo1_2th_text_reg
3/22/17 5:21 PM  W/com.door43.translationstudio.core.TargetTranslation: Missing manifest file in target translation en-x-demo1_2th_text_reg
     

```

--- a/app/src/main/java/com/door43/translationstudio/ui/newtranslation/NewTargetTranslationActivity.java
+++ b/app/src/main/java/com/door43/translationstudio/ui/newtranslation/NewTargetTranslationActivity.java
@@ -16,14 +16,11 @@ import org.json.JSONException;
 import org.unfoldingword.door43client.models.Questionnaire;
 import org.unfoldingword.door43client.models.TargetLanguage;
 import org.unfoldingword.resourcecontainer.Project;
-import org.unfoldingword.resourcecontainer.Resource;
-import org.unfoldingword.resourcecontainer.ResourceContainer;
 import org.unfoldingword.tools.logger.Logger;
 
 import com.door43.translationstudio.App;
 import com.door43.translationstudio.R;
 import com.door43.translationstudio.core.MergeConflictsHandler;
-import com.door43.translationstudio.core.Migration;
 import com.door43.translationstudio.core.TranslationViewMode;
 import com.door43.translationstudio.tasks.MergeTargetTranslationTask;
 import com.door43.translationstudio.ui.SettingsActivity;
@@ -44,7 +41,6 @@ import org.unfoldingword.tools.taskmanager.SimpleTaskWatcher;
 import org.unfoldingword.tools.taskmanager.TaskManager;
 
 import java.io.IOException;
-import java.util.List;
 
 public class NewTargetTranslationActivity extends BaseActivity implements TargetLanguageListFragment.OnItemClickListener, ProjectListFragment.OnItemClickListener, SimpleTaskWatcher.OnFinishedListener {
 
@@ -320,15 +316,7 @@ public class NewTargetTranslationActivity extends BaseActivity implements Target
 //            SourceLanguage sourceLanguage = App.getLibrary().getPreferredSourceLanguage(projectId, App.getDeviceLanguageCode()); // get project name
             // TODO: 3/2/2016 eventually the format will be specified in the project
 
-            Project p = App.getLibrary().index().getProject(App.getDeviceLanguageCode(), projectId, true);
-            List<Resource> resources = App.getLibrary().index().getResources(p.languageSlug, p.slug);
-            ResourceContainer resourceContainer = null;
-            try {
-                resourceContainer = App.getLibrary().open(p.languageSlug, p.slug, resources.get(0).slug);
-            } catch (Exception e) {
-                e.printStackTrace();
-            }
-            TranslationFormat format = TranslationFormat.parse(resourceContainer.contentMimeType);
+            TranslationFormat format = projectId.equals("obs") ? TranslationFormat.MARKDOWN : TranslationFormat.USFM;
             final TargetTranslation targetTranslation = App.getTranslator().createTargetTranslation(App.getProfile().getNativeSpeaker(), mSelectedTargetLanguage, projectId, ResourceType.TEXT, resourceSlug, format);
             if(targetTranslation != null) {
                 // deploy custom language code request to the translation

--- a/app/src/main/res/layout-port/toolbar_translator_sidebar.xml
+++ b/app/src/main/res/layout-port/toolbar_translator_sidebar.xml
@@ -43,6 +43,16 @@
             android:src="@drawable/ic_view_week_inactive_24dp"
             android:background="?attr/selectableItemBackground"
             android:layout_gravity="center" />
+
+        <ImageButton
+            android:id="@+id/warn_merge_conflict"
+            android:layout_width="wrap_content"
+            android:layout_height="fill_parent"
+            android:padding="16dp"
+            android:src="@drawable/ic_warning_inactive_24dp"
+            android:background="?attr/selectableItemBackground"
+            android:layout_gravity="center"
+            android:visibility="gone"  />
     </LinearLayout>
 
     <!-- middle -->

